pkg/framework/test/etcd.go

```diff
@@ -18,11 +18,10 @@ type Etcd struct {
 	session        *gexec.Session
 	stdOut         *gbytes.Buffer
 	stdErr         *gbytes.Buffer
-	dataDirManager DataDirManager
+	dataDirManager dataDirManager
 }
 
-// DataDirManager knows how to create and destroy Etcd's data directory.
-type DataDirManager interface {
+type dataDirManager interface {
```

Comment:
Or you can move that to an internal packages if you want to reuse it somewhere else.